database/tables/news_items.pg

```diff
@@ -0,0 +1,15 @@
+columns
+    date: timestamp with time zone not null default now()
+    directory: text not null
+    for_students: boolean not null default false
+    id: bigint not null default nextval('news_items_id_seq'::regclass)
+    order_by: integer not null default 0
```

Comment:
Is it necessary to have an `order_by` when it seems natural to order by `date`?

Reply:
Maybe the column name could be better -- the sort order used is actually `date, order_by`. The need for `order_by` is to disambiguate cases where we have two items with the same date. An example of this is the current news items (two of them) which will both be inserted into the DB with the same date field. I wanted the sort order to be stable, which is why I added `order_by`.